Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/StorageProviderCommands/Factories/QueryCommandFactoryTest.cs

```diff
@@ -173,7 +174,7 @@ public void CreateForScalarQuery ()
       var commandBuilderStub = new Mock<IDbCommandBuilder>();
       var expectedParametersWithType = GetExpectedParametersForQueryStub();
       _dbCommandBuilderFactoryStrictMock
-          .Setup(stub => stub.CreateForQuery("statement", expectedParametersWithType))
+          .Setup(stub => stub.CreateForQuery(QueryStatementType.Text, "statement", expectedParametersWithType))
```

Comment:
I think we might want to inject this as a test parameter with values Text and StoredProc for each of the scalar, collection, and custom queries to show that the StatementType is passed through instead of hardcoded. We could isntead also simply use different values in each test method, but it seems like why shouldn't we use parameterized tests given they are not much more complicated anyway.